### PR TITLE
docs: use repo-metadata to generate README

### PR DESCRIPTION
**Pull request by JustinBeckwith**

### CODE REVIEW

File: samples/README.md [hunk 1]
```diff
@@ -1,45 +1,170 @@
...
 
 ## Table of Contents
 
 * [Before you begin](#before-you-begin)
 * [Samples](#samples)
-  * [Create Virtual Machine with Apache and Custom Homepage](#create-virtual-machine-with-apache-and-custom-homepage)
-  * [List Virtual Machines](#list-virtual-machines)
+  * [Create VM](#create-vm)
+  * [Delete VM](#delete-vm)
+  * [List V Ms](#list-v-ms)
```
Comment (bcoe) [inline samples/README.md]: likewise, we providing some metadata should fix this weirdness.

File: README.md [hunk 2]
```diff
@@ -37,80 +39,85 @@ Google APIs Client Libraries, in [Client Libraries Explained][explained].
...
+| Create VM | [source code](https://github.com/googleapis/nodejs-compute/blob/master/samples/createVM.js) | [![Open in Cloud Shell][shell_img]](https://console.cloud.google.com/cloudshell/open?git_repo=https://github.com/googleapis/nodejs-compute&page=editor&open_in_editor=samples/createVM.js,samples/README.md) |
+| Delete VM | [source code](https://github.com/googleapis/nodejs-compute/blob/master/samples/deleteVM.js) | [![Open in Cloud Shell][shell_img]](https://console.cloud.google.com/cloudshell/open?git_repo=https://github.com/googleapis/nodejs-compute&page=editor&open_in_editor=samples/deleteVM.js,samples/README.md) |
+| List V Ms | [source code](https://github.com/googleapis/nodejs-compute/blob/master/samples/listVMs.js) | [![Open in Cloud Shell][shell_img]](https://console.cloud.google.com/cloudshell/open?git_repo=https://github.com/googleapis/nodejs-compute&page=editor&open_in_editor=samples/listVMs.js,samples/README.md) |
```
[the remark this answers is not in the document]
Comment (JustinBeckwith) [reply README.md]: Good call - added!

File: samples/README.md (hunk 1 above)
Comment (JustinBeckwith) [reply samples/README.md]: Fixed :)

File: README.md [hunk 3]
```diff
@@ -93,7 +93,7 @@ has instructions for running the samples.
...
 | Delete VM | [source code](https://github.com/googleapis/nodejs-compute/blob/master/samples/deleteVM.js) | [![Open in Cloud Shell][shell_img]](https://console.cloud.google.com/cloudshell/open?git_repo=https://github.com/googleapis/nodejs-compute&page=editor&open_in_editor=samples/deleteVM.js,samples/README.md) |
-| List V Ms | [source code](https://github.com/googleapis/nodejs-compute/blob/master/samples/listVMs.js) | [![Open in Cloud Shell][shell_img]](https://console.cloud.google.com/cloudshell/open?git_repo=https://github.com/googleapis/nodejs-compute&page=editor&open_in_editor=samples/listVMs.js,samples/README.md) |
+| List VMs | [source code](https://github.com/googleapis/nodejs-compute/blob/master/samples/listVMs.js) | [![Open in Cloud Shell][shell_img]](https://console.cloud.google.com/cloudshell/open?git_repo=https://github.com/googleapis/nodejs-compute&page=editor&open_in_editor=samples/listVMs.js,samples/README.md) |
```
Comment (bcoe) [inline README.md]: Nice :)

**Pull request merged**